rest-api/data_gen/fake_participant_generator.py

```diff
@@ -709,6 +704,26 @@ def generate_participant(self, include_physical_measurements, include_biobank_or
         last_request_time = max(last_request_time, last_hpo_change_time)
       self._submit_status_changes(participant_id, last_request_time)
 
+  def add_pm_and_biospecimens_to_participants(self, participant_id_list):
+    self._force_measurement = True
```

Comment:
nit: Using persistent state here doesn't seem ideal as it could affect subsequent method calls. Would be better to just accept another optional param to the relevant methods.